### ISSUE TITLE
Clarify ORCID vs ORCID iD

### ISSUE THREAD

**Issue by JacquemotMC** [RDA-DMP-Common/RDA-DMP-Common-Standard#82]
Should we use ORCID or ORCID iD as type?

**Comment by JacquemotMC** [comment]
Yes I meant the value for the identifier type. There was a recommendation to tu use "ORCID id" but I guess the value "ORCID" is more widely spread.

**Comment by MarekSuchanek** [comment]
OK, but what would you change e.g. in the table here: https://github.com/RDA-DMP-Common/RDA-DMP-Common-Standard/blob/master/README.md#properties-in-contributor_id so it is correct with the terminology "ORCID iD"?
